text/text.md

```diff
@@ -72,50 +72,57 @@ OK, jungle. And you tell tall tales around the fire, right?
 And you want to hear mine? 
 OK, Duke, I owe you. So here it goes. 
 
+#####The transition between this first paragraph and the second one is a bit rough. The character talks to a person named Duke and then it suddenly goes to Gummy. While it's established that there are multiple people the narrator is talking to, it's a bit confusing as there is a jarring jump.
 What do you mean, Gummy? It's the plain truth. I will tell it to you exactly like 
 it happened.  Yes, sir, I'm sure, what are you, a fuckin' people's court?  
 Do I have to take the oath or what?
 
 Alright, then. This is the real story of how I came to be in the
 possession of Manuel the Man-Made Man. It starts in Mobile, which is
-only fitting, given the circus obsession with phrases with words that
-all start the same. Mobile, yes, time ago it was in Alabama, but now
-it is a rough and tumble harbor town close to the border of the United
+only fitting, given the circus obsession with phrases that
+all start the same.  Ah, Mobile. Long ago it was in Alabama, but now
+it is a rough-and-tumble harbor town close to the border of the United
 All Races People's Republic of Florida. That's a pretty long name for
 a place that starts at Pascagoula in the West and and goes no further
-than Brunswick in the East, if only our people have their way. Which
+than Brunswick in the East, <i>if only our people have their way. Which
 they will, I have no doubt as a loyal citizen of the Republic.  Every
 dawning day brings new brothers to our new republic. Brothers of all
-races, creeds and dispositions.
-
+races, creeds and dispositions.</i>
+#####The last part where I italicized is a bit confusing.  What people? What way? I think if you maybe explained it a little more or put a transition here (like a sentence), it would make the part less confusing.
```

Comment:
Well, it's the *People's* republic. Maybe add a sentence here emphasizing that it's the republic of *our* people. This sentence is important, because it tries to assert the character as a loyal citizen of the republic.